io-engine/src/grpc/v1/pool.rs

```
@@ -709,39 +710,23 @@ impl PoolRpc for PoolService {
         .await
     }
 
+    async fn grow_pool(&self, _request: Request<GrowPoolRequest>) -> GrpcResult<GrowPoolResponse> {
+        unimplemented!()
```

Let's return an error here